nservicebus/hosting/ibus-instance.md

```diff
@@ -0,0 +1,113 @@
+---
```

Comment:
why do we need a new page? adding this doco here http://docs.particular.net/nservicebus/containers/injecting-ibus should suffice